README.md

```diff
@@ -23,7 +23,7 @@ Create the virtualenv:
 
 Install the requirements:
 
-    pip install -r requirements.txt
+    sudo pip install -r requirements.txt
```

Comment:
of course I will merge all the other updated :))
you should submit a corrected pull request, but for this time I can approve this pull request 
and remove the sudo pip from the README in another commit in master.
thank you.